background.js

```diff
@@ -112,8 +115,30 @@ const MyQOnly = {
     }
   },
 
+  async githubReviewRequests(username) {
```

Comment:
Good idea to factor this out into its own function. I'll probably want to do this with the other two as well (you don't need to do that).